marimo/_plugins/stateless/mpl/_mpl.py

```diff
@@ -509,3 +509,17 @@ def dispose(self, context: RuntimeContext, deletion: bool) -> bool:
     filter: invert(0.3);
 }
 """.strip()
+
+
+def patch_javascript(javascript: str) -> str:
```

Comment:
I'm guessing we are doing this because we don't want to vendor this file into marimo?